### PR TITLE
added main data processing file

### PR DESCRIPTION
**Pull request by VictorGillioz**
This PR adds a main file to run data processing from mesh files to .pt files. The loop and saving dynamic is moved from utils to this main processing file.

### CODE REVIEW

File: meshnets/process.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+flags.DEFINE_bool(
+    'verbose', False,
+    'Wether or not the data processing should log processing information.')
+
+
+def main(_):
+    """Loop through a directory containing mesh files, produce a
+    graph representation of each mesh and save the graph as
+    a `.pt` file in the processed data directory."""
```
Comment (fabiocruz) [inline meshnets/process.py:39]: One liner + longer description.

File: meshnets/process.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+FLAGS = flags.FLAGS
+
+# Path flags
+flags.DEFINE_string('mesh_data_dir', os.path.join('data', 'vtk'),
+                    'Path to the folder containing the mesh files.')
+flags.DEFINE_string('processed_data_dir', os.path.join('data', 'pt'),
+                    'Path to the folder for the processed data files.')
+
+flags.DEFINE_bool('get_pressure', True,
+                  'Wether or not to extract pressure from the mesh files.')
```
Comment (fabiocruz) [inline meshnets/process.py:29]: Typo, "W**h**ether**.

File: meshnets/process.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    if not os.path.exists(processed_data_dir):
+        os.makedirs(processed_data_dir)
+
+    for mesh_file in os.listdir(mesh_data_dir):
+
+        mesh_file_path = os.path.join(mesh_data_dir, mesh_file)
+        processed_graph = data_processing.mesh_file_to_graph_data(
+            mesh_file_path,
+            WIND_VECTOR,
+            get_pressure=get_pressure,
```
Comment (fabiocruz) [inline meshnets/process.py:55]: In which circumstance would you not want to get the pressure? Should the logic behind the data loading functions allow selecting whether or not to load velocity **and** whether or not to load pressure?

**Review by fabiocruz — APPROVED** [review]
LGTM! Left a question about the rationale for the data loading functions, all else is minor.

File: meshnets/process.py (hunk 3 above)
Comment (IvanPombo) [reply meshnets/process.py:55]: The logic with this is that I am providing the mesh with already the pressure in. Ideally, one can provide one input file with just the mesh, and another with the same mesh and pressure measurements. But as of now, I am just providing the latter on, and he can extract the input from this.

Comment (VictorGillioz) [reply meshnets/process.py:55]: This also allows to process data for pure inference, where no pressure has been computed and is thus not present in the file.

Comment (fabiocruz) [reply meshnets/process.py:57]: Right, so the graph with spatial coordinates as node features is always loaded and pressure is only loaded when **get_pressure=True**?

Comment (VictorGillioz) [reply meshnets/process.py:57]: Exactly. Would you find this clearer if I change the argument to `load_pressure`?

Comment (fabiocruz) [reply meshnets/process.py:57]: Got it. No need to rename the argument, I just wanted to understand what was going in this data loading function.

**Pull request merged**